docs/templates.md

```diff
@@ -0,0 +1,24 @@
+# Distributed Application Templates
+
+Distributed Applications (dApps) are basically decentralized applications. The templates can be found [here](https://github.com/scale-it/algo-builder-templates) and are extremely helpful and easy to use. They are designed and implemented in such a way that they can provide the aspiring developers a headstart in building dApps based on `Algorand Blockchain`. The templates heavily use [algosdk](https://github.com/algorand/js-algorand-sdk), [algo-builder](https://github.com/scale-it/algo-builder) and [AlgoSigner](https://github.com/PureStake/algosigner).
+
+## Usage
+
+An `algob` project and an `react` frontend are encapsulated in each template. Deployment information is stored in `checkpoints` (in `/artifacts`) from which user can import information directly (loading `.yaml` file in react-app).
```

Comment:
Please add how to use on top line, you can add description below as a note, 
also in usage please explain how to unbox with different examples i.e which word is which, folder name or template name

Reply:
ok

Reply:
@amityadav0 This is explained below. In the `Quick Start` section.

Reply:
ok, i don't think this line should be put in usage, i think usage is about how to use

Reply:
we can put this in note below, or above in some section

Reply:
Maybe we can update the title to `Template Structure/Composition`. As the lines below explain the basic components of any template (`algob project`, `reactjs fe`, `/artifacts`..).
BDW - We also have this info in the `algo-builder-templates` readme, we could just link as well.

Reply:
Good idea